db/migrate/20220117034711_create_dependabot_alerts.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDependabotAlerts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :dependabot_alerts do |t|
+      t.string :name
+      t.string :description
+      t.string :severity
+      t.string :status
```

Comment:
Forgot to mention but we'll also have to change the primary key for Dependabot Alerts to be a string, and call it `node_id`. I'm not exactly sure how to do this but should be achievable with some Googling ;)

Reply:
Ok so have the name and the node_id, or just the node_id?

Reply:
yeah both. node_id is gonna be like a random hash of characters